omnichain/minter/tasks/deploy.ts

```diff
@@ -14,21 +12,19 @@ const main = async (args: any, hre: HardhatRuntimeEnvironment) => {
   const [signer] = await hre.ethers.getSigners();
   console.log(`🔑 Using account: ${signer.address}\n`);
 
-  const SYSTEM_CONTRACT = getAddress("systemContract", hre.network.name);
-  const bitcoinChainID = 18332;
+  const systemContract = getAddress("systemContract", "zeta_testnet");
 
-  const factory = await hre.ethers.getContractFactory(contractName);
+  const factory = await hre.ethers.getContractFactory("Minter");
   const contract = await factory.deploy(
     "Wrapped tBTC",
     "WTBTC",
-    bitcoinChainID,
-    SYSTEM_CONTRACT
+    18332,
```

Comment:
why did you remove the const? I prefer the const, make it easier to understand

Reply:
So that there are fewer changes to the template. I can revert this.

Reply:
f9cd800